contracts/examples/guards/DebugTransactionGuard.sol

```diff
@@ -24,13 +26,29 @@ contract DebugTransactionGuard is BaseGuard {
         Enum.Operation operation,
         uint256 safeTxGas,
         bool usesRefund,
-        uint256 nonce
+        uint256 nonce,
+        bytes signatures,
+        address executor
     );
 
     event GasUsage(address indexed safe, bytes32 indexed txHash, uint256 indexed nonce, bool success);
 
     mapping(bytes32 => uint256) public txNonces;
 
+    /**
+     * @notice Called by the Safe contract before a transaction is executed.
+     * @param to Destination address of Safe transaction.
+     * @param value Ether value of Safe transaction.
+     * @param data Data payload of Safe transaction.
+     * @param operation Operation type of Safe transaction.
+     * @param safeTxGas Gas that should be used for the Safe transaction.
+     * @param baseGas Gas costs that are independent of the transaction execution(e.g. base transaction fee, signature check, payment of the refund)
+     * @param gasPrice Gas price that should be used for the payment calculation.
+     * @param gasToken Token address (or 0 if ETH) that is used for the payment.
+     * @param refundReceiver Address of receiver of gas payment (or 0 if tx.origin).
+     * @param signatures Packed signature data ({bytes32 r}{bytes32 s}{uint8 v})
```

Comment:
Should "tx.origin" and "{bytes32 r}{bytes32 s}{uint8 v}" be in backticks?

Reply:
I reused signatures description from https://github.com/safe-global/safe-contracts/pull/396